libbeat/common/transport/tlscommon/tls.go

```diff
@@ -69,11 +72,17 @@ func LoadCertificate(config *CertificateConfig) (*tls.Certificate, error) {
 
 // ReadPEMFile reads a PEM format file on disk and decrypt it with the privided password and
```

Comment:
Seems the function name + docs should be updated to adjust for the new behaviour?